presto-main/src/main/java/com/facebook/presto/execution/StageInfo.java

```diff
@@ -76,11 +77,16 @@ public URI getSelf()
     }
 
     @JsonProperty
-    public StageExecutionInfo getLatestAttemptExecutionInfo()
+    public Optional<StageExecutionInfo> getLatestAttemptExecutionInfo()
     {
         return latestAttemptExecutionInfo;
     }
 
+    public List<TaskInfo> getLatestTasks()
```

Comment:
nit: let's move it after the `@JsonProperty` getters (before the `isFinalStageInfo`)